src/models/streaming/AudioConfig.ts

```diff
@@ -0,0 +1,22 @@
+export default class AudioConfig {
+    contentType: string;
+    layout?: string;
+    rate?: number;
+    format?: string;
+    channels?: number;
+    constructor(contentType: string, layout?: string, rate?: number, format?: string, channels?: number){
+        this.contentType = contentType;
+        this.layout = layout;
+        this.rate = rate;
+        this.format = format;
+        this.channels = channels;
+    }
+
+    getContentTypeString (): string {
+        return `${this.contentType}` +
+            (this.layout == null ? "" : `;layout=${this.layout}`) +
```

Comment:
do you have to check for `null`? Can `this.layout` be `undefined` as well? Would a check like `this.layout ? "" : `;layout=${this.layout}`` work as well?

Reply:
It can be null, == covers both of these cases whereas === would only cover if it was exactly null. I'll see if your suggestion works, that would be cleaner

Reply:
I know we can do `!this.layout ?` for sure

Reply:
It works. I wasn't sure if `this.layout` on its own would handle undefined checking but it makes sense that it does